fees-rewards.md

```diff
@@ -0,0 +1,77 @@
+# Fees and Rewards
+
+Obscuro sets out a simple structure for fees and rewards. End users pay fees based on their usage which are placed into a pool from which rewards for nodes are paid out.
+
+The structure delivers a steady and predictable income for node operators over the long term and a predictable fee for users proportional to the actual costs of running the network and does not rely on bidding wars or vary according to activity.
+
+The structure also doesn&#39;t punish Aggregators that miss out on a publishing opportunity with slashing. Instead, the loss of reward should be incentive enough.
+
+## Cost of L2 nodes
+
+Operators run the network and incur operational costs by running and maintaining nodes.
+
+An &quot;expected monthly node operational costs&quot; variable is set through the governance protocol, which is designed to reflect how much a node operator would pay for hardware, hosting etc, in a month. This is analogous to the proof of work difficulty found in other blockchains. This effects demand for running a node.
+
+Aggregator nodes incur cost from submitting transactions to L1, through the following:
+
+- Storing rollup data on Ethereum.
+- The overhead in participating in publishing rollups outside of the storage cost, e.g. signing transactions.
+
+## Fee per transaction
+
+The total fee a user must pay has three components.
+
+1. A proportion of the monthly operational cost for all nodes. The proportion is calculated based on the computation of the current transaction as a proportion of the moving average of all L2 transactions over the month. Represented as:
+
+![equation_1.png](images/equation_1.png)
+
+![equation_2.png](images/equation_2.png)
+
+2. The cost incurred by nodes for L1 storage, represented as:
+
+![equation_2.png](images/equation_3.png)
+
+3. A proportion of the additional overhead cost incurred by nodes for processing rollups is as follows:
+
+The idea behind this is:
+
+- At the point of transaction, it&#39;s impossible to know what the true cost of the future rollup is
+- Users need to pay their fair share of what this future rollup cost will be
+- The best indicator of what the future rollup cost might be is a moving average of the previous n blocks
```

Comment:
The best indicator or an indicator? Statisticians might argue this is not the best

Reply:
"Previous n blocks" or "previous n rollups"?